src/dotnet-ildasm/Infrastructure/TypeReferenceExtensions.cs

```diff
@@ -5,22 +5,27 @@ namespace DotNet.Ildasm.Infrastructure
 {
     public static class TypeReferenceExtensions
     {
-        public static string ToILType(this TypeReference typeReference)
+        public static string ToILNameFormatt(this TypeReference typeReference)
```

Comment:
I would not rename this just yet. Once the IL generation is done, it may make sense to get the full logic extracted from this code base.